docs/typescript/how-to-send-a-signal-from-a-client-in-typescript.md

```diff
@@ -1,8 +1,8 @@
 ---
-id: how-to-send-a-signal-from-a-workflow-in-typescript
-title: How to send a Signal from Workflow in TypeScript
-sidebar_label: Send a Signal from Workflow
-description: Send a Signal from Workflow
+id: how-to-send-a-signal-from-a-client-in-typescript
```

Comment:
This is an issue that should be fixed separately - so we can get it through now if the warning is still popping up.